central/scannerdefinitions/handler/handler.go

```diff
@@ -248,7 +262,7 @@ func (h *httpHandler) getUpdater(t updaterType, key string) *requestedUpdater {
 	return updater
 }
 
-func (h *httpHandler) handleScannerDefsFile(ctx context.Context, zipF *zip.File) error {
+func (h *httpHandler) handleScannerDefsFile(ctx context.Context, zipF *zip.File, blobName string) error {
```

Comment:
Is there a way to increase the timeout of POST request to upload file? I notice the upload can take quite a few seconds if not minutes.

Reply:
It's already a flag:

```
$ roxctl scanner upload-db --help
Upload the vulnerability database for StackRox Scanner and/or Scanner V4.

Usage:
  roxctl scanner upload-db [flags]

Flags:
  -h, --help                     help for upload-db
      --scanner-db-file string   File containing the dumped Scanner definitions DB
  -t, --timeout duration         timeout for API requests; represents the maximum duration of a request (default 10m0s)
```